### ISSUE TITLE
isAuthenticated always returns true

### ISSUE THREAD

**Issue by jovermier** [nhost/nhost#328]
After clearing site cache and reloading the page isAuthenticated still returns true.

"@nhost/react-apollo": "4.0.5"

```
  const userData = useUserData();
  const isAuthenticated = useAuthenticated();
  console.log('userData:', userData);
  console.log('isAuthenticated:', isAuthenticated);
```

Output:
```
userData: undefined
isAuthenticated: true
```

**Comment by plmercereau** [comment]
Hello,
Can't reproduce with `@nhost/react-apollo@4.0.8` released today.
- could you try upgrading to latest package versions?
- just to be sure, does clearing cache also include resetting localStorage? In Chrome, it is usually done when choosing "clear site data"

**Comment by jovermier** [comment]
Seems to be resolved in 4.0.8
